robots/preservation/transfer_object.rb

```diff
@@ -0,0 +1,89 @@
+
+module Preservation
+  # Robot for transferring objects from the DOR export area to the Moab object deposit area
+  class TransferObject < Base
+    def initialize(opts = {})
+      super(REPOSITORY, WORKFLOW_NAME, 'transfer-object', opts)
```

Comment:
I didn't make 'transfer-object' a constant because it is only used once.  It might make sense to make it a constant anyway.  What do you think?  e.g.

```ruby
ROBOT_NAME = 'transfer_object'.freeze
```

Reply:
sounds good!